OpenOversight/app/templates/index.html

```diff
@@ -70,11 +68,16 @@ <h1>
                 </h1>
                 <p>
                    While Seattle Tech Bloc does not accept donations at this time, if you'd like to financially
```

Comment:
Should we update to Orca Collective?

Reply:
Nah, I think here we can keep it as Seattle Tech Bloc. The domain & contact information all correspond to that still.